akka-persistence/src/test/scala/akka/persistence/PersistentActorRecoveryTimeoutSpec.scala

```diff
@@ -15,7 +15,7 @@ object PersistentActorRecoveryTimeoutSpec {
     SteppingInmemJournal.config(PersistentActorRecoveryTimeoutSpec.journalId).withFallback(
       ConfigFactory.parseString(
         """
-          |akka.persistence.journal.stepping-inmem.recovery-event-timeout=100ms
+          |akka.persistence.journal.stepping-inmem.recovery-event-timeout=500ms
```

Comment:
That's what I was thinking, but it's just a single negative test case so maybe a second wouldn't hurt too much.